ttnn/cpp/ttnn/operations/conv/conv2d/device/conv2d_op.cpp

```diff
@@ -92,9 +96,27 @@ Tensor optimized_conv_new(const Tensor& a, const Tensor &b, std::optional<const
                 auto output_layout = untilize_out ? Layout::ROW_MAJOR : Layout::TILE;
                 auto arch = is_tensor_on_device_or_multidevice(a) ? a.device()->arch() : ttnn::operations::experimental::auto_format::AutoFormat::GetDefaultDevice()->arch();
                 bool fp32_accum = a.device()->arch() == tt::ARCH::WORMHOLE_B0;  // && compute_kernel_config.has_value()) ? compute_kernel_config.value().fp32_dest_acc_en : false;
+                auto optimized_conv_op =  OptimizedConvNew(sliding_window_config,
+                                  output_channels,
+                                  groups,
+                                  untilize_out,
+                                  bias.has_value(),
+                                  fuse_relu,
+                                  parallelization_config,
+                                  block_config,
+                                  memory_config,
+                                  dtype,
+                                  input_tensor_shape,
+                                  use_shallow_conv_variant,
+                                  compute_kernel_config,
+                                  enable_act_double_buffer,
+                                  enable_weights_double_buffer,
+                                  enable_split_reader,
+                                  enable_subblock_padding,
+                                  use_non_tile_height);
+                optimized_conv_op.pre_op_l1_allocation_size_bytes = stats.total_allocated_bytes;
```

Comment:
Since you only need stats.total_allocated_bytes, it's better to capture just that int in lambda

Reply:
Done.